### PR TITLE
Add ESLint and Prettier

### PR DESCRIPTION
**Pull request by some-molasses**
This PR adds ESLint and Prettier to the repository, to help maintain a consistent format as we bring in new web devs.  It then reformats existing files to Prettier's standards.

(also takes some carousel functions out of global scope, but maintains the functionality).

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (ad-world) [inline .prettierrc.json:1]: is this supposed to be empty?

File: server/api/admin-services/contact-us.ts [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
-import { ReadWriteController } from '../read-write-controller';
+import { ReadWriteController } from "../read-write-controller";
 
 export class ContactUsAdminService {
   // Updates the name of each MathSoc exec
   static updateExecs(req: Request, res: Response): void {
     const formData = req.body;
 
-    const currentContactUs = require('../../data/contact-us.json');
+    // this is likely to get overwritten by the JSON editor anyway
```
Comment (ad-world) [inline server/api/admin-services/contact-us.ts:10]: yea possibly in a future pr, lets rework this into using the editor

File: .prettierrc.json (hunk 1 above)
Comment (some-molasses) [reply .prettierrc.json:1]: weirdly, yes it is actually

It tells the IDE to use prettier I think?

**Review by ad-world — APPROVED** [review]
Just fix merge conflicts before merging

**Review by ad-world — APPROVED** [review]

**Pull request merged**